### PR TITLE
fix(curriculum) replace single-line blocks with multi-line blocks for…

### PR DESCRIPTION
**Pull request by LaurentLabine**
… Front End Libraries #41523

Second attempt

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Checks Front End Libraries on #41518 - Second Attempt

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ dashedName: use--for-a-more-concise-conditional
 
 The `if/else` statements worked in the last challenge, but there's a more concise way to achieve the same result. Imagine that you are tracking several conditions in a component and you want different elements to render depending on each of these conditions. If you write a lot of `else if` statements to return slightly different UIs, you may repeat code which leaves room for error. Instead, you can use the `&&` logical operator to perform conditional logic in a more concise way. This is possible because you want to check if a condition is `true`, and if it is, return some markup. Here's an example:
 
-`{condition && <p>markup</p>}`
+```js
+{condition && <p>markup</p>}
+```
```
Comment (nhcarrigan) [inline curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md:15]:
```suggestion
```jsx
{condition && <p>markup</p>}
```
```
Mirroring @RandellDawson's suggestion from the previous PR.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Almost there. We have just one more outstanding change, I believe.

I've created a suggestion - if you click the "Commit Suggestion" button, GitHub's UI will give you an option to commit the change directly which should resolve the issue.

**Comment by LaurentLabine** [comment]
Thanks! Made the change and hopfeully it'll be good to go...

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me. Thank you for all of your hard work on this. 🙂

**Review by RandellDawson — APPROVED** [review]
@LaurentLabine Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by LaurentLabine** [comment]
I was also thinking I wouldn't mind doing more of these especially for the mess I've caused but I've also read the first-timers contribution piece and wouldn't want to be taking away somebody else's shot at what seems to be a pretty good chance so let me know what you think is best!

**Comment by RandellDawson** [comment]
> I was also thinking I wouldn't mind doing more of these especially for the mess I've caused but I've also read the first-timers contribution piece and wouldn't want to be taking away somebody else's shot at what seems to be a pretty good chance so let me know what you think is best!

This one is not labeled for `first-timers`, so if you want to do another PR for the issue, let us know on the issue, so we can assign you to it.

**Pull request merged**